app/script/conversation/ConversationError.coffee

```diff
@@ -28,6 +28,8 @@ class z.conversation.ConversationError
     @message = switch @type
       when z.conversation.ConversationError::TYPE.CONVERSATION_NOT_FOUND
         'Conversation not found'
+      when z.conversation.ConversationError::DEGRADED_CONVERSATION_CANCELATION
```

Comment:
Sorry for being picky: CANCELATION -> CANCELLATION in both BE and AE
`when z.conversation.ConversationError::DEGRADED_CONVERSATION_CANCELLATION`